examples/parameter_demo/main.cpp

```diff
@@ -11,14 +11,15 @@
 
 #include "log.h"
```

Comment:
in commit ` lib: utils: move FNV-1a key hash from parameter to utils ` message, you explain the what but not the why of this commit. A sentence is enough, but thats the reason of the commit that matters.

Reply:
Every commits has been reworded.